parmed/gromacs/gromacstop.py

```diff
@@ -1364,7 +1364,7 @@ def write(self, dest, combine=None, parameters='inline'):
                 if print_atnum:
                     parfile.write('%8d ' % atom_type.atomic_number)
                 parfile.write('%10.5f  %10.6f  A %13.6g %13.6g\n' % (
-                              atom_type.mass, 0, atom_type.sigma/10,
+                              atom_type.mass, atom_type.charge, atom_type.sigma/10,
```

Comment:
If you change `atom_type.charge` to `atom_type.charge or 0.0` that will handle the case where `atom_type.charge` is `None`